tools/prune-old-builds.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+
+# Copyright (C) 2020  Matthew "strager" Glazar
+# See end of file for extended copyright information.
+
+import os
+import sys
+import shutil
+import argparse
+import subprocess
+from pathlib import Path
+from datetime import datetime
+
+
+FOURTEEN_DAYS = 14
+
+
+def how_old(date: datetime) -> int:
+    """Return a integer that represents how many days has passed given a date"""
+    return (datetime.now() - date).days
+
+
+def error_print(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
+
+
+def get_commits(repo_name: str, repository_url: str) -> list:
+    try:
+        return subprocess.check_output(
+            f"cd {repo_name} \
+            && git fetch --prune {repository_url} '+refs/pull/*/head:refs/remotes/github-pr/*' \
+            && git rev-list --all --remotes",
+            stderr=subprocess.STDOUT, shell=True
+        ).decode('utf-8').split('\n')
+    except subprocess.CalledProcessError as err:
+        error_print(err.output.decode())
+        exit(err.returncode)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser("Prune old builds")
+    parser.add_argument(
+        "builds_path",
+        help="The path where the builds are stored.",
+        nargs=1,
+        type=str)
+    parser.add_argument(
+        "repository_url",
+        help="The url to the repository.",
+        nargs=1,
+        type=str)
+    parser.add_argument("-f", "--delete", action="store_true",
+                        help="Delete folders.")
+    args, _ = parser.parse_known_args()
+
+    builds_path = Path(args.builds_path[0])
+    # https://github.com/quick-lint/quick-lint-js.git
+    repository_url = args.repository_url[0]
```

Comment:
Nit: `nargs` causes `argparse` to give you lists. If you omit `nargs`, you get just the `str`. Remove `nargs`:
```suggestion
    parser.add_argument(
        "builds_path",
        help="The path where the builds are stored.",
        type=str)
    parser.add_argument(
        "repository_url",
        help="The url to the repository.",
        type=str)
    parser.add_argument("-f", "--delete", action="store_true",
                        help="Delete folders.")
    args, _ = parser.parse_known_args()

    builds_path = Path(args.builds_path)
    # https://github.com/quick-lint/quick-lint-js.git
    repository_url = args.repository_url
```